invenio_stats/tasks.py

```diff
@@ -11,9 +11,26 @@
 
 from celery import shared_task
 from dateutil.parser import parse as dateutil_parse
+from datetime import timedelta
 
 from .proxies import current_stats
 
+StatsEventTask = {
```

Comment:
Style change! I'm happy to revert it if you don't agree. I thought it would be nicer to define the task in the tasks module. Then, whatever module wants to use it would just have to include this definition